Math/NumberTheory/Zeta/Dirichlet.hs

```diff
@@ -0,0 +1,212 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- |
+-- Module:      Math.NumberTheory.Zeta.Dirichlet
+-- Copyright:   (c) 2018 Andrew Lelechenko
```

Comment:
You have every right to credit yourself in copyrights, if you write module from the scratch.

Reply:
Alright, then if it is not a problem I'll change it in the `Pentagonal` module as well.

Reply:
Absolutely, go ahead.